src/main/java/com/pnudev/springonlinedelivery/OrderU.java

```diff
@@ -0,0 +1,33 @@
+package com.pnudev.springonlinedelivery;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Getter;
+
+@Entity
+@Table(name = "orderU")	
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+public class OrderU {
```

Comment:
If I understand correctly, this is an intermediate table for linking users and orders (or menu items). If that is the case, then you should consider this: 
You don't need to create intermediate tables in JPA, JPA itself should be able to create them automatically if you specify annotations for both ends of relation (`MenuItem` and `User` in your case). Refer to your ERD for suitable relation type between those entites (probably many-to-many but I might be wrong), check https://vladmihalcea.com/the-best-way-to-use-the-manytomany-annotation-with-jpa-and-hibernate/ or other as a reference

Reply:
Here is my model that explains everything on the spot: https://dbdiagram.io/d/6415caa0296d97641d88f2fd
Since order is a reserved word in MySQL, I decided to name the class this way (OrderU), although it could have gone back to Order.

Reply:
Got it. You may want to use backticks (``"`order`"``) or escape quotes (`"\"order\""`)